clang/include/clang/Basic/Attr.td

```diff
@@ -107,6 +107,10 @@ def NonBitField : SubsetSubject<Field,
                                 [{!S->isBitField()}],
                                 "non-bit-field non-static data members">;
 
+def BitField : SubsetSubject<Field,
+                             [{S->isBitField()}],
+                             "bit-field non-static data members">;
```

Comment:
I don't know how much this text helps here, the diagnostic specifying 'non static' seems a little silly.  The `NonBitField` one is important because it applies to non-static data members that ARE NOT bitfields.  But in this case, 'must be a bitfield' also implies 'non static'.

Reply:
Fixed